cleanlab_pinecone_rag_workflow/Cleanlab_Pinecone_RAG.ipynb

```diff
@@ -0,0 +1,2673 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "98708651",
+   "metadata": {},
+   "source": [
+    "Cleanlab Studio runs many analyses of your dataset to find data issues and provide other useful metadata, including detection of: label issues, outliers, ambiguous examples, well-labeled examples, and more. For this particular use case, we will focus on text-specific data issues that Cleanlab can detect. After Cleanlab Studio runs these analyses on your data, new metadata columns are available to show the issue types detected and the severity of the issue type.\n",
+    "\n",
+    "We will first ingest the document chunks data obtained from our Pinecone DB into Cleanlab Studio and then run a project to analyze this data and look for the text issue types mentioned. After the project results are ready, we can make use of the `download_cleanlab_columns()` column to download the added metadata columns from the Cleanlab Studio project results.\n",
+    "\n",
+    "For more details on Cleanlab Studio Metadata you can go [here](https://help.cleanlab.ai/guide/concepts/cleanlab_columns/)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 143,
+   "id": "629b7d20-8e85-4476-94fe-c00c90a56f94",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cleanlab_studio import Studio\n",
+    "\n",
+    "API_KEY = \"YOUR API KEY HERE\"\n",
+    "studio = Studio(API_KEY)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b7be24fe-070b-4879-8345-4993b052d958",
+   "metadata": {},
+   "source": [
+    "### Ingest our documents chunks into Cleanlab Studio"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 144,
+   "id": "70da45d4-b0e6-4aed-9ff5-702bd9341538",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Uploading dataset...: 100%|██████████|\n",
+      "Ingesting Dataset...: 100%|██████████|"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Dataset ID: 1169f187757e43079b5cc9b7d93b482e\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "dataset_id = studio.upload_dataset(\n",
+    "    chunk_df,\n",
+    "    dataset_name=\"cleanlab_pinecone_RAG_document_chunks\"\n",
+    ")\n",
+    "print(f\"Dataset ID: {dataset_id}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b5fc6210-7949-42c8-84ff-c2e56578ba9d",
+   "metadata": {},
+   "source": [
+    "### Launch a Cleanlab Studio Project using our documents data to analyze the text"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 145,
+   "id": "ed715976-dad7-497c-94df-cbe2f4ba48e4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "text_column = \"text\""
```

Comment:
Nit: Why do we do `text_column = "text"` ? We also pass in `    modality="text"`...